components/hab/src/command/studio/enter.rs

```diff
@@ -132,10 +133,29 @@ mod inner {
         return false;
     }
 
-    fn rerun_with_sudo_if_needed(ui: &mut UI) -> Result<()> {
-        // If I have root permissions, early return, we are done.
+    fn has_docker_group() -> bool {
+        let groupname = "docker".to_string();
```

Comment:
You can drop the call to `to_string()` here, since the only place you use this variable is on line 139, below, and that function call expects a string slice.  You could also drop this line completely if you wish, and just pass `"docker"` to `get_members_by_groupname` directly.